packages/elements/src/tree/helpers/renderer.ts

```diff
@@ -11,49 +14,59 @@ type RendererScope = {
   noRelation?: boolean;
 };
 
+export const createTreeRenderer = <T extends TreeDataItem = TreeDataItem>(
+  scope?: unknown
+): ((item: T, composer: CollectionComposer<T>, element?: HTMLElement) => HTMLElement) => {
```

Comment:
This looks unnecessary.
```suggestion
) => {
```